functions/New-DbaDbUser.ps1

```diff
@@ -0,0 +1,247 @@
+		.PARAMETER Confirm
+			If this switch is enabled, you will be prompted for confirmation before executing any operations that change state.
+		.PARAMETER EnableException
+			By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
+			This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
+			Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+
+		.NOTES
+			Tags: Database, User
+			Author: Frank Henninger (@osiris687)
+			Website: https://dbatools.io
+			Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
+			License: GNU GPL v3 https://opensource.org/licenses/GPL-3.0
+		.LINK
+			https://dbatools.io/New-DbaDbUser
+
+		.EXAMPLE
+		New-DbaDbUser -SqlInstance sqlserver2014 -Database DB1 -Login user1
+		Creates a new sql user with login named user1 in the specified database
+
+		.EXAMPLE
+		New-DbaDbUser -SqlInstance sqlserver2014 -Database DB1 -Username user1
+		Creates a new sql user without login named user1 in the specified database
+
+		.EXAMPLE
+		Get-DbaDatabaseUser -SqlInstance sqlserver1 -Database DB1 | New-DbaDbUser -SqlInstance sqlserver2 -Database DB1
+		Copies users from sqlserver1.DB1 to sqlserver2.DB1. Does not copy permissions!
+
+	#>
+    [CmdletBinding(SupportsShouldProcess = $true, DefaultParameterSetName = "NoLogin")]
+    param (
+        [parameter(Mandatory, Position = 1)]
+        [Alias("ServerInstance", "SqlServer")]
+        [DbaInstanceParameter[]]$SqlInstance,
+        [PSCredential]$SqlCredential,
+        [Alias("Databases")]
+        [object[]]$Database,
+        [object[]]$ExcludeDatabase,
+        [switch]$IncludeSystem,
+        [Alias("Name", "LoginName")]
+        [parameter(ParameterSetName = "Login")]
+        [string[]]$Login,
+        # [parameter(ParameterSetName = "Contained")]
+
+        [parameter(ParameterSetName = "Login")]
+        [parameter(ParameterSetName = "NoLogin")]
+        [string[]]$Username,
+        # [parameter(ParameterSetName = "Contained")]
+
+        # [Security.SecureString]$Password,
+
+        [switch]$Force,
+        [switch][Alias('Silent')]
+        $EnableException
+    )
+    
+    begin {
+        
+        
+    }
+    process {
+        foreach ($instance in $SqlInstance) {
+            try {
+                Write-Message -Level Verbose -Message "Connecting to $instance"
+                $server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $sqlcredential
+            }
+            catch {
+                Stop-Function -Message "Failure" -Category ConnectionError -ErrorRecord $_ -Target $instance -Continue
+                return
+            }
+            
+            $databases = $server.Databases | Where-Object Status -EQ "normal"
```

Comment:
This needs to be based on `IsAccessible`, the status of normal won't include databases that are not accessible due to read-only access or involved in mirroring and such.